.scalafmt.conf

```diff
@@ -0,0 +1,23 @@
+version = "2.0.1"
+maxColumn = 120
+align = most
```

Comment:
This one is actually kinda controversal. E.g. it could generate smth like this:
`val instance: Config              = ConfigFactory.load(commonConfigPath).getConfig(commonConfigNamespace)`
Should we keep it? Or mb just set it to `none`? I want to discuss this.

Reply:
I would propose to remove it

Reply:
I think that the best option is `more`, bc it will align only pattern-matching arrows